src/containers/Login/component.js

```diff
@@ -19,44 +21,46 @@ const Login = ({
             sm={{ span: 10, offset: 1 }}
             lg={{ span: 8, offset: 2 }}
           >
-            <Form onFinish={login}>
+            <Form onSubmit={handleSubmit}>
               <Typography.Title>The Movie DB</Typography.Title>
               {errorTxt && (
-              <Alert
-                type="error"
-                message={errorTxt}
-              />
+                <Alert
+                  type="error"
+                  message={errorTxt}
+                />
               )}
-              <Form.Item name="username">
-                <Input
-                  prefix={(
+              <Field
+                name="username"
+                component={InputField}
+                placeholder="Username"
+                prefix={
+                  (
                     <UserOutlined
                       style={{ color: 'rgba(0,0,0,.25)' }}
                     />
-                  )}
-                  placeholder="Username"
-                />
-              </Form.Item>
-              <Form.Item name="password">
-                <Input
-                  prefix={(
+                  )
+                }
+              />
+              <Field
+                name="password"
+                component={InputField}
+                placeholder="Password"
+                type="password"
+                prefix={
+                  (
                     <LockOutlined
                       style={{ color: 'rgba(0,0,0,.25)' }}
                     />
-                  )}
-                  type="password"
-                  placeholder="Password"
-                />
-              </Form.Item>
-              <Form.Item>
-                <Button
-                  loading={loading}
-                  type="primary"
-                  htmlType="submit"
-                >
-                  Log in
-                </Button>
-              </Form.Item>
+                  )
+                }
+              />
+              <Button
+                loading={loading}
+                type="primary"
+                htmlType="submit"
+              >
+              Log in
```

Comment:
Looks like broken indents, could you enable linter on CI (GitHub Actions)?

Reply:
fixed & added GitHub Actions